libtelemetry/src/main/java/com/mapbox/android/telemetry/AppUserTurnstile.java

```diff
@@ -5,28 +5,39 @@
 import android.os.Parcel;
 import android.os.Parcelable;
 
+import androidx.annotation.Keep;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import com.google.gson.annotations.SerializedName;
 
 import static com.mapbox.android.telemetry.TelemetryEnabler.TELEMETRY_STATES;
 
+@Keep
 public class AppUserTurnstile extends Event implements Parcelable {
   private static final String APP_USER_TURNSTILE = "appUserTurnstile";
   private static final String OPERATING_SYSTEM = "Android - " + Build.VERSION.RELEASE;
   private static final String APPLICATION_CONTEXT_CANT_BE_NULL = "Create a MapboxTelemetry instance before calling "
     + "this method.";
 
+  @SerializedName("event")
```

Comment:
`@SerializedName` will be obsolete if you use `@Keep`

Reply:
@SerializedName is added for safety reasons and consistency. 
Still keeping @SerializedName annotation to make sure this does not malfrom the event again in case this class is extended by some other top level class and @Keep annotation is forgotten.

Reply:
> `@SerializedName` will be obsolete if you use `@Keep`